frontend/src/hooks/useHashURL.js

```diff
@@ -22,12 +22,6 @@ export const useHashURL = (defaultKey) => {
     const location = useLocation()
     const currentKey = location.hash.slice(1)
     const setCurrentKey = useCallback((key) => { history(`#${key}`) }, [])
-    // If no key is currently in the url then add the default key automatically.
```

Comment:
This was messing up the history and also generating a React warning in the console, so I got rid of it.